src/Pages/CheckOutPage.tsx

```diff
@@ -24,6 +24,11 @@ export function CheckOut() {
         }  
     }
 
+    function onDeleteHandler(index: number) {
+        cartData.splice(index, 1)
+        setCartData([...cartData])
```

Comment:
```suggestion
        setCartData([...cartData])
        let price = cartData[index].quantity * cartData[index].product.product_price;
        setCartTotal(cartTotal - price);
```

Reply:
Great job finding out how to update this on delete. When testing the only thing I noticed was that the price is not updating on deletion. Adding these two lines made the price update as well.